src/v/redpanda/admin/api-doc/broker.json

```diff
@@ -24,6 +24,72 @@
                     "parameters": []
                 }
             ]
+        },
+        {
+            "path": "/v1/brokers/{id}/membership_state",
```

Comment:
can this state can be folded into the existing model used to return per-broker information used for `GET v1/brokers` (see above) and the same model then can be returned for `GET v1/brokers/{id}` instead of an entirely separate URL dedicated for the broker membership state?

Reply:
yes, I will fold this